stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -229,12 +229,12 @@ spec:
             - sh
             - -ec
             - >-
+              /bin/mongodb_exporter
             {{- if .Values.auth.enabled }}
-              export MONGODB_URL=mongodb://$METRICS_USER:$METRICS_PASSWORD@localhost:{{ .Values.port }};
```

Comment:
Why the change to connect to admin database for auth enabled? Think it should be as before.

Reply:
done https://github.com/helm/charts/pull/10026/commits/429615119febde083ea72f182d71d9467182c5e0
seems that `admin` is default one